examples/angular/src/app/pages/form/form.component.ts

```diff
@@ -69,15 +68,11 @@ export class FormComponent implements OnInit {
     });
   }
 
-  onSubmit() {
-    if (!this.userForm.valid) {
-      this.sixFormUtil.focusInvalidField();
-    }
+  onSubmit(event: SubmitEvent) {
+    console.log('form submitted', event);
```

Comment:
Are log entries getting removed during the build?
A debug mode is always great, but it might be a bit of an annoyance for a productive build.

Reply:
No, the log entries are not removed, but as this is only the demo app, I'd rather keep them in.